src/v/redpanda/application.cc

```diff
@@ -652,7 +652,7 @@ void application::start() {
     if (coproc_enabled()) {
         /// Temporarily disable retries for the new client until we create a
         /// more granular way to configure this per client or per request.
-        kafka::client::shard_local_cfg().retries.set_value(size_t(0));
+        kafka::client::shard_local_cfg().retries.set_value(size_t(1));
```

Comment:
Every error is a "normal situation" for the proxy, I don't think there's a consistent set of "normal situations".

I think I'll add an optional retries parameter to each request.

At some point I'd like to have a think about cancellation in general; there's probably a concept here that would allow expiry of a request based on: a timeout, a retry limit, a bytes transferred limit, etc.